test/regression.bzl

```diff
@@ -27,14 +34,14 @@ def regression_test(
             #
             # https://bazel.build/reference/be/common-definitions#test.size
             size = _pop(kwargs, "size", "small"),
-            timeout = _pop(kwargs, "timeout", "moderate"),
             srcs = ["//test:bazel_test.sh"],
             args = [],
             data = [
                 "//:openroad",
                 "//test:regression_resources",
                 "//test:regression_test.sh",
             ] + test_files + _pop(kwargs, "data", []),
+            env_inherit = ["PATH"],  # To find system binaries such as `getconf`
```

Comment:
Does this imply that we are giving up sandboxing in the tests?

Reply:
Well, they never could be sandboxed to begin with as right now, the tcl scripts call unexpected programs  such as `getconf` (typically, on a regular system it might just work, but on my  stricter system (NixOS), it became apparent).

If this is something we need more often, maybe we should swig a tcl function that returns the value of `std::thread::hardware_concurrency`

Reply:
If the only need is to get the number of cpus we could easily add that.  Are there any other programs?

Reply:
I don't know as this was the first it failed on.

Reply:
Can/should we define nix environment to define what system installed tools bazel builds can/should rely on and hook it up in CI?

At what point do caching of builds/tests break?